src/test/java/org/b612foundation/adam/opm/OdmFormatterTest.java

```diff
@@ -10,6 +10,13 @@ public void testParseSimpleOpmHappy() throws Exception {
     OrbitParameterMessage parsed = OdmFormatter.parseOpmString(OdmScenarioBuilder.getSimpleOpm());
     Assert.assertEquals(parsed, expected);
   }
+  
+  @Test
+  public void testParseSimpleOpmTTHappy() throws Exception {
+    OrbitParameterMessage expected = OdmScenarioBuilder.buildSimpleOpmTT();
```

Comment:
Rather than adding to OdmScenarioBuilder (which is quite widely used), maybe better to build a regular OPM, then set the time system here - then it's easy to test multiple time systems.